src/main/java/com/emotion/emotiondesktop/Helper/HttpResponse.java

```diff
@@ -0,0 +1,14 @@
+package com.emotion.emotiondesktop.Helper;
+
+import lombok.Getter;
+
+@Getter
+public class HttpResponse {
+    private final String responseBody;
+    private final int responseCode;
+
+    public HttpResponse(String responseBody, int responseCode) {
+        this.responseBody = responseBody;
+        this.responseCode = responseCode;
+    }
+}
```

Comment:
Remember about adding empty line at the end of the file - look for other files, because it is not only place